doc/source/general.rst

```diff
@@ -3,9 +3,12 @@ Code Overview
 
 The Qibo framework in this repository implements a common system to deal with classical hardware and future quantum hardware.
 
-The code abstraction is located in ``qibo/base``. The default simulation engine is implemented using tensorflow and is located in ``qibo/tensorflow``.
+The code abstraction is located in ``qibo/base`` and the core simulation engine is located in ``qibo/core``.
```

Comment:
Just to simplification, shall we rename `base` in `abstractions` or `interfaces`?

Reply:
I renamed `base` to `abstractions`, `BaseCircuit` to `AbstractCircuit` and `backends.base.BaseBackend` to `backends.abstract.AbstractBackend`.